tests/test_modeling_common.py

```diff
@@ -274,6 +275,21 @@ def test_attention_outputs(self):
                     [self.model_tester.num_attention_heads, decoder_seq_length, decoder_key_length],
                 )
 
+                # cross attentions
+                cross_attentions = outputs.cross_attentions
+                self.assertIsInstance(cross_attentions, (list, tuple))
+                self.assertEqual(len(cross_attentions), self.model_tester.num_hidden_layers)
+                self.assertListEqual(
+                    list(cross_attentions[0].shape[-3:]),
+                    [
+                        self.model_tester.num_attention_heads,
+                        (self.model_tester.ngram + 1) * decoder_seq_length
```

Comment:
since this is just a prophetnet case, I would consider overwriting the full test in `test_modeling_prophetnet.py`. or at least adding a comment. The `model_tester.ngram` conditional does not make it clear that it's only a `prophetnet` thing.

Reply:
dang it - I thought no-one would notice haha -> you're right, I should this function in ProphetNet. It'd make this common test much easier to read